packages/permissions/src/utils/has-permissions.js

```diff
@@ -69,3 +70,14 @@ export const hasSomePermissions = (demandedPermissions, actualPermissions) =>
   demandedPermissions.some(permission =>
     hasPermission(permission, actualPermissions)
   );
+
+// Returns an Array<String> of unconfigured (not passed as `demandedPermissions`) permissions.
+// The shapes of the arguments are:
+// - demandedPermissions:
+//     ['ViewProducts', 'ManageOrders']
+// - actualPermissions:
+//     { canViewProducts: true, canManageOrders: false }
+export const getInvalidPermissions = (demandedPermissions, actualPermissions) =>
```

Comment:
@emmenko referring to (you commented on an outdated ref I can't respond on):

> How about simply getInvalidPermissions?

Sure, I don't mind. Invalid can entail more than unconfigured to it may be more flexible for future needs.